core/src/test/java/eu/fasten/core/data/metadatadb/MetadataDaoTest.java

```diff
@@ -366,24 +400,26 @@ public void insertMultipleDependenciesTest() throws IllegalArgumentException {
         var packageId = 1L;
         var dependencyIds = Arrays.asList(8L, 42L);
         var versionRanges = Arrays.asList(new String[]{"1.0.0-1.9.9"}, new String[]{"2.1.0-2.1.9"});
-        var insertValues = Mockito.mock(InsertValuesStep3.class);
+        var metadata = Arrays.asList(new JSONObject(), new JSONObject());
+        var insertValues = Mockito.mock(InsertValuesStep4.class);
         Mockito.when(context.insertInto(Dependencies.DEPENDENCIES, Dependencies.DEPENDENCIES.PACKAGE_VERSION_ID,
-                Dependencies.DEPENDENCIES.DEPENDENCY_ID, Dependencies.DEPENDENCIES.VERSION_RANGE)).thenReturn(insertValues);
-        Mockito.when(insertValues.values(packageId, dependencyIds.get(0), versionRanges.get(0))).thenReturn(insertValues);
-        Mockito.when(insertValues.values(packageId, dependencyIds.get(1), versionRanges.get(1))).thenReturn(insertValues);
+                Dependencies.DEPENDENCIES.DEPENDENCY_ID, Dependencies.DEPENDENCIES.VERSION_RANGE, Dependencies.DEPENDENCIES.METADATA)).thenReturn(insertValues);
+        Mockito.when(insertValues.values(packageId, dependencyIds.get(0), versionRanges.get(0), JSONB.valueOf("{}"))).thenReturn(insertValues);
+        Mockito.when(insertValues.values(packageId, dependencyIds.get(1), versionRanges.get(1), JSONB.valueOf("{}"))).thenReturn(insertValues);
```

Comment:
The additional parameters are unrelated to the clean-up, are they? What changed here?

Reply:
@MihhailSokolov

Reply:
There was an error in the test. This test was built to test the functionality of the insertion of 3 attributes but recently we added the fourth attribute but forgot to update the test. So that's what was done here.